### PR TITLE
Address deprecations from doctrine/dbal

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -53,6 +53,11 @@ parameters:
             message: '~^Call to function method_exists\(\) with Doctrine\\Migrations\\Metadata\\Storage\\MetadataStorage and ''getSql'' will always evaluate to true\.$~'
             path: lib/Doctrine/Migrations/Version/DbalExecutor.php
 
+        # TODO: deprecate using the connection event manager and expose
+        # our own event manager instead.
+	-   message: '~^Call to deprecated method getEventManager\(\) of class Doctrine\\DBAL\\Connection\.$~'
+            path: lib/Doctrine/Migrations/DependencyFactory.php
```
Comment (greg0ire) [inline phpstan.neon.dist:60]: This one should be addressed on 3.6.x, because I think it will involve a deprecation.

Comment (stof) [reply phpstan.neon.dist:60]: indeed. Using a separate EventManager will require consumers to change how they register their listener.

Comment (greg0ire) [reply phpstan.neon.dist:60]: I created a separate issue to track this: https://github.com/doctrine/migrations/issues/1287

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you!

**Pull request merged**